lms/templates/provider/authorize.html

```diff
@@ -0,0 +1,55 @@
+{% extends "main_django.html" %}
+
+{% load scope i18n %}
+
+{% block bodyclass %}oauth2{% endblock %}
+
+{% block body %}
+<div class="authorization-confirmation">
+  {% if not error %}
+  {% blocktrans with application_name=client.name %}
+  <p><strong>{{ application_name }}</strong> would like to access your data with the following permissions:</p>
+  {% endblocktrans %}
+  <ul>
+    {% for permission in oauth_data.scope|scopes %}
+    <li>
+      {% if permission == "openid" %}
+        {% trans "Read your user ID" %}
+      {% elif permission == "profile" %}
+        {% trans "Read your user profile" %}
+      {% elif permission == "email" %}
+        {% trans "Read your email address" %}
+      {% elif permission == "course_staff" %}
+        {% trans "Read the list of courses in which you are a staff member." %}
+      {% elif permission == "course_instructor" %}
+        {% trans "Read the list of courses in which you are an instructor." %}
+      {% elif permission == "permissions" %}
+        {% trans "To see if you are a global staff user" %}
+      {% else %}
+        {% blocktrans %}Manage your data: {{ permission }}{% endblocktrans %}
+      {% endif %}
+    </li>
+    {% endfor %}
+  </ul>
+  <form method="post" action="{% url "oauth2:authorize" %}">
+    {% csrf_token %}
+    {{ form.errors }}
+    {{ form.non_field_errors }}
```

Comment:
Screenshots of errors.

![error2](https://cloud.githubusercontent.com/assets/3853/12920518/f82cfaea-cf16-11e5-9f8f-7ccd3fc3e95f.png)
![error](https://cloud.githubusercontent.com/assets/3853/12920519/f840dfb0-cf16-11e5-8147-57bc80090891.png)